CMake/kwiver-depends-GDAL.cmake

```diff
@@ -0,0 +1,31 @@
+# Optionally find and configure GDAL dependency
+
+option( KWIVER_ENABLE_GDAL
+  "Enable GDAL dependent code and plugins (Arrows)"
+  ${fletch_ENABLE_GDAL}
```

Comment:
@chetnieter if this is not turning on this option by default, then maybe Fletch is not exporting `fletch_ENABLE_GDAL`